docs/DesignDocs/SGX_QuoteVerify_Integration.md

```diff
@@ -1,30 +1,33 @@
-Attestation: OE SDK Integration with Intel® SGX DCAP Quote Verification Library for SGX Evidence verification
+Attestation: OE SDK Integration with Intel® SGX DCAP Quote Verification Library for SGX Evidence Verification
 ====
 
-This design document proposes an extension of the OE SDK implementation
+Note: please format the text lines to be within 80 characters if possible.
```

Comment:
The current content causes vscode to be overwhelmed, and very slow in processing edit input. Not sure if the the long lines and the big block of code are the culprit, or is it a bug in vscode.